### PR TITLE
my patches

### PR DESCRIPTION
**Pull request by zelurker**
Well as I said on the forum you wouldn't want to merge all these patches, especially the one on magic experience, but you are the one who wanted a pull request, so here is one...

### CODE REVIEW

File: apps/openmw/mwphysics/physicssystem.cpp [hunk 1]
```diff
@@ -1152,9 +1152,9 @@ namespace MWPhysics
 
                 bool waterCollision = false;
                 if (effects.get(ESM::MagicEffect::WaterWalking).getMagnitude()
-                        && cell->getCell()->hasWater()
-                        && !world->isUnderwater(iter->first.getCell(),
-                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
+                        && cell->getCell()->hasWater())
+//                        && !world->isUnderwater(iter->first.getCell(),
+//                                               osg::Vec3f(iter->first.getRefData().getPosition().asVec3())))
```
Comment (psi29a) [inline apps/openmw/mwphysics/physicssystem.cpp]: It is better to remove code then to comment it out, if we need it, we can always go back in time with git to retrieve this.

**Comment by psi29a** [comment]
@scrawl can you also eyeball this please :)

**Comment by scrawl** [comment]
Please create a separate branch and pull request for each topic. I have a few comments, but I will wait for separate PRs before posting them.

File: apps/openmw/mwphysics/physicssystem.cpp (hunk 1 above)
Comment (dteviot) [inline apps/openmw/mwphysics/physicssystem.cpp]: The waterCollision variable is never used. So lines 1153 to 1158 can be removed.

Comment (dteviot) [inline apps/openmw/mwphysics/physicssystem.cpp]: I'm wrong.  Variable is used.
For some reason, when I did a search for waterCollision, only the two instances were shown.
D'oh.

**Pull request closed without merging**